docs/reference/continuous-export.md

```diff
@@ -0,0 +1,169 @@
+  "lastKeysInPrefixRegexp": [
+    // list of regexps of keys to exported last in each prefix (for signalling) (optional)
+    ".*/_SUCCESS"
+  ]
+}
+```
+Once set on a branch, every commit or merge to that branch will be exported.
+
+#### Operation
+
+Export current branch state just once, without continuous operation, with `POST` to
+`/repositories/{repository}/branches/{branch}/export-hook`.
+
+Resume export after repairing with a `POST` to
+`/repositories/{repository}/branches/{branch}/repair-export`.
+
+## Operation
+
+When continuous export is enabled for a branch its current state is exported to S3.  From
+then on, every commit or merge operation to that branch will be exported.  The commit or
+merge operation on the branch returns as soon as export starts.  Multiple commits to the
+same exported branch may cause some otherwise-overlapping exports to be skipped.
+Eventually the last export will appear.  This is in line with "eventual consistency" of
+S3.
+
+The export process runs for a while and copies all files.  This process is highly parallel and
+does not maintain the creation or modification orders of the source files.  Export can write
+specific status files _after_ all other files in a directory. Set `lastKeysInPrefixRegexp` to
+match directories that need a status file. lakeFS export will create an empty file
+`_lakefs_success` in each matching directory.  These status files are written *last* in their
+S3 prefix, allowing triggers based on S3 write ordering to succeed.
+
+Export can additionally write a status file to the export status path.  If set, lakeFS will
+write a file `<REPO>-<BRANCH>-<COMMIT>` to that directory when done exporting.  It looks like:
+> ``` > status: <STATUS>, signalled_errors: <NUM-ERRORS> > ```
+
+The status will be one of:
+
+* `exported-successfully`
+* `export-failed`
+* `export-repaired`
+
+S3 does not offer atomic operations across objects.  Exported objects read from S3 remain
+valid only while the above `_STATUS` file shows `"success"` and the `"commit"` is unchanged!
+
+## FAQs
+
+#### What happens if a continuously exported branch changes while a previous export is still in progress?
+
+lakeFS exports objects safely and will not export the same branch twice concurrently.
+However when multiple concurrent exports occur intermediate exports cannot be observed and
+may be dropped.  A dropped intermediate export does not create the `_SUCCESS` or
+`_STATUS_<timestamp>.json` and `_MANIFEST_<timestamp>.json` files; conversely, once these
+appear the export will run to completion.
+
+#### How consistent is export once the `_STATUS_<timestamp>.json` file appears?
+
+Once the `_STATUS_<timestamp>.json` file holds status `success`, the export has finished,
+and S3 rules apply:
+
+* Any file can be accessed.
+* Files names appear in the `_MANIFEST_<timestamp>.json` file.
+* However S3 file listings will only eventually show all accessible files.
```

Comment:
No longer true

Reply:
Yup.  Also the filenames have changed :-)